java/code/src/com/suse/manager/webui/controllers/ImageBuildController.java

```diff
@@ -828,7 +830,11 @@ private static JsonObject getImageInfoSummary(ImageOverview imageOverview) {
         }
 
         imageOverview.getBuildServerAction()
-                .ifPresent(ba -> json.addProperty("statusId", ba.getStatus().getId()));
+                .ifPresentOrElse(ba -> json.addProperty("statusId", ba.getStatus().getId()),
+                                 () -> json.addProperty("statusId",
+                                            imageOverview.isBuilt() ? ActionFactory.STATUS_COMPLETED.getId() :
+                                                                      ActionFactory.STATUS_FAILED.getId())
```

Comment:
Not sure here, but wouldn't this display failed action for images still building?

Reply:
This is just a fallback when the build action does not exist. This can happen when buildhost is removed etc.
During building, the build action exists and the status is taken from it.